CPR/FaellesSQLCprService.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+/*
+ * This file is part of aakb/kontrolgruppen-core-bundle.
+ *
+ * (c) 2019 ITK Development
+ *
+ * This source file is subject to the MIT license.
+ */
+
+namespace Kontrolgruppen\CoreBundle\CPR;
+
+use Kontrolgruppen\CoreBundle\Entity\Client;
+use Symfony\Contracts\HttpClient\Exception\TransportExceptionInterface;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+
+class FaellesSQLCprService implements CprServiceInterface
+{
+    private const CITIZEN_ENDPOINT = 'citizen';
+
+    private $serviceUrl;
+    private $httpClient;
+
+    public function __construct(HttpClientInterface $httpClient, $serviceUrl)
+    {
+        $this->serviceUrl = $serviceUrl;
+        $this->httpClient = $httpClient;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function find(int $cpr): array
+    {
+        $url = sprintf('%s/%s/%s', $this->serviceUrl, self::CITIZEN_ENDPOINT, $cpr);
+
+        try {
+            $response = $this->httpClient->request('GET', $url);
+
+            return json_decode($response->getContent(), true);
+        } catch (TransportExceptionInterface $e) {
+            throw new CprException($e->getMessage());
```

Comment:
I prefer to make the original exception available in the custom exception, e.g. use something like `new CprException($e->getMessage(), $e->getCode(), $e)`.